ui/js/models/perf/alerts.js

```diff
@@ -95,8 +97,8 @@ treeherder.factory('PhAlerts', [
                 }
                 resultStr += "Summary of tests that regressed:\n\n" +
                              _.map(regressed, function(alert) {
-                                 return "  " + alert.title + ": " + alert.prev_value + " -> " +
-                                 alert.new_value + " (" + alert.amount_pct + "% worse)";
+                                 return String(" " + alert.amount_pct.toFixed(2) + "%  " ).slice(-8) + alert.title + ": " + alert.prev_value + " -> " +
```

Comment:
Two things:

* I don't think we want sub-percents, so I'd use toFixed(0) here (and below)
* The left padding you use here will break if we have a regression larger than 99% (since then we'll want to use 2 or more characters of padding). Could you use lodash's padLeft function to get deterministic behavior here? https://github.com/lodash/lodash/blob/3.10.1/doc/README.md#_padleftstring-length0-chars You will probably need to figure out the largest number of digits in the alert percentages in advance for this to work.